src/v1/index.ts

```diff
@@ -328,3 +341,114 @@ function decodeBallotVotes(contests: Contests, bits: BitReader): VotesDict {
 
   return votes
 }
+
+interface HMPBBallotPageMetadata {
```

Comment:
This is almost the same as `BallotPageMetadata` from hmpb-interpreter. Does it make sense to consolidate somehow?

Reply:
I'm thinking `hmpb-interpreter` should use this data structure in a future refactor?